app/src/main/java/io/appium/uiautomator2/handler/GetElementAttribute.java

```diff
@@ -56,7 +79,90 @@ public AppiumResponse safeHandle(IHttpRequest request) {
         } catch (UiAutomator2Exception e) {
             Logger.error(MessageFormat.format("Unable to retrive attribute {0}", attributeName), e);
             return new AppiumResponse(getSessionId(request), WDStatus.UNKNOWN_ERROR, e);
+        } catch (ReflectiveOperationException | InvalidSelectorException e) {
+            Logger.error("Can not access to method or field: ", e);
+            return new AppiumResponse(getSessionId(request), WDStatus.UNKNOWN_ERROR, e);
+        }
+
+    }
+
+    private class ContentSize {
+        int width;
+        int height;
+        int top;
+        int left;
+        int scrollableOffset;
+        int touchPadding;
+
+        ContentSize(Rect rect) {
+            width = rect.width();
+            height = rect.height();
+            top = rect.top;
+            left = rect.left;
+        }
+
+        @Override
+        public String toString() {
+            JSONObject jsonObject = new JSONObject();
+            try {
+                jsonObject.put("width", width);
+                jsonObject.put("height", height);
+                jsonObject.put("top", top);
+                jsonObject.put("left", left);
+                jsonObject.put("scrollableOffset", scrollableOffset);
+                jsonObject.put("touchPadding", touchPadding);
+            } catch (JSONException e) {
+                e.printStackTrace();
+            }
+            return jsonObject.toString();
+        }
+    }
+
+    private int getScrollableOffset(AndroidElement uiScrollable) throws UiObjectNotFoundException, ClassNotFoundException, InvalidSelectorException {
+        AccessibilityNodeInfo nodeInfo = null;
+        int offset = 0;
+        if (uiScrollable instanceof UiObject) {
+            UiObject object = (UiObject) uiScrollable.getChild(new UiSelector().index(0));
```

Comment:
this might fail if the element has no children

Reply:
it will fail with an error we catch, but the message there isn't good. will fix.